src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -739,6 +740,23 @@ public function testTimeoutOnAccess()
         $response->getHeaders();
     }
 
+    public function testTimeoutIsNotAFatalError()
+    {
+        $client = $this->getHttpClient(__FUNCTION__);
+        $response = $client->request('GET', 'http://localhost:8057/timeout-body', [
+            'timeout' => 0.3,
```

Comment:
Explanation why were these values changed would be nice

Reply:
I'm going to backport the changes to 4.4 once the test are not transient anymore.